spec/controllers/sitemap_controller_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper'
+
+describe SitemapController do
+
+  describe 'GET #index' do
+
+    let :static_pages_urls do
+      [root_url, about_url, contact_url, faq_url, login_url, signup_url]
+    end
+
+    it 'renders the sitemap template' do
+      get :index, format: :xml
+      expect(response).to render_template(:index)
+    end
+
+    it 'assigns @static_pages' do
+      get :index, format: :xml
+      expect(assigns(:static_pages)).to eql(static_pages_urls)
+    end
+
+    it 'assigns @courses' do
+      course = [FactoryGirl.create(:course)]
```

Comment:
It would probably be better in a let block so it can be lazily evaluated.

We are trying to transition away from factory girl in our unit tests because of how slow they make the test suite. You had no possible way of knowing that though. I need to write up some guidelines for the tests etc. 

Anyway I would replace this with a double, something like this should work:
```
let(:course) { double('Course') }

before do
   allow(Course).to receive(:all).and_return(course)
end
```